networkx/readwrite/gml.py

```diff
@@ -153,12 +153,12 @@ def read_gml(path, label='label', destringizer=None):
 
     label : string, optional
         If not None, the parsed nodes will be renamed according to node
-        attributes indicated by ``label``. Default value: ``'label'``.
+        attributes indicated by `label`. Default value: `'label'`.
```

Comment:
Well,

    ``'label'``

will render like `'label'`, and

     `'label'`

will render like *'label'*, I think. The former indicates a code snippet, which clarifies that the quotes are part of the code. The latter doesn't make it clear whether the quotes are part of the code or not.

You can see the effect in, for example, the Sphinx documentation for the [default role][1] configuration option: the string `'py:obj'`, including the quotes, is formatted as code.

[1]: http://www.sphinx-doc.org/en/stable/config.html#confval-default_role